src/Database/NeoServer.Data/Configurations/AccountModelConfiguration.cs

```diff
@@ -86,6 +86,15 @@ private static void Seed(EntityTypeBuilder<AccountModel> builder)
                 Password = "1",
                 PremiumTime = 30,
                 AllowManyOnline = true
+            },
```

Comment:
why creating a second account for another GOD ?

Reply:
I neeed test with multiple account for kick another player. 

Sample:
    Account 1[GOD] -> kick -> account 2